pkg/tool/crypto/aes.go

```diff
@@ -51,6 +51,10 @@ func getAESKey() string {
 	return aesKey
 }
 
+func GetAesKey() string {
```

Comment:
If we want `getAESKey()` to be exported, change the original function other than add a new function.